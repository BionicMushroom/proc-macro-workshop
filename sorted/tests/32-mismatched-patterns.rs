#[sorted::check]
fn main() {
    const X: u32 = 0;
    let x = 0;

    #[sorted]
    match x {
        X => println!("0"),
        1 => println!("1"),
        _ => println!("other"),
    }
}
