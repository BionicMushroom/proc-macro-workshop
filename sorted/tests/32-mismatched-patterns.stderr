error: comparison between this pattern and the previously encountered patterns is unsupported by #[sorted]
 --> tests/32-mismatched-patterns.rs:9:9
  |
9 |         1 => println!("1"),
  |         ^
